src/main/io/ledstrip.h

```diff
@@ -33,10 +33,14 @@
 #define LED_CONFIGURABLE_COLOR_COUNT   16
 #define LED_MODE_COUNT                  6
 #define LED_DIRECTION_COUNT             6
-#define LED_BASEFUNCTION_COUNT          7
+#define LED_BASEFUNCTION_COUNT          10
 #define LED_OVERLAY_COUNT               7
 #define LED_SPECIAL_COLOR_COUNT        11
 
+#define LED_BAR_GPS 0
+#define LED_BAR_BATTERY 1
+#define MAX_BAR_STATES 2
```

Comment:
```suggestion
enum ledBarIds {
   LED_BAR_GPS ,
   LED_BAR_BATTERY,
   LED_BAR__COUNT
};
```

Also, the is no need to make this public Move it into ledstrip.c